include/cudaq/Support/TargetConfig.h

```diff
@@ -24,6 +24,8 @@ enum TargetFeatureFlag : unsigned {
   flagsFP64 = 0x0002,
   flagsMgpu = 0x0004,
   flagsMqpu = 0x0008,
+  flagsDepAnalysis = 0x0010,
+  flagsQPP = 0x0020,
```

Comment:
Rather than exposing a bitset to the user (which I assume is what this does?), we should think about using strings/names in a comma separate list to make these option comprehensible to mortals.

Reply:
> Rather than exposing a bitset to the user (which I assume is what this does?),

That is not what this does.

> we should think about using strings/names in a comma separate list to make these option comprehensible to mortals.

This is what it does. See Adam's new test files to see how this is used.

Reply:
OK, good.